spec/exchanges/novaexchange/integration/market_spec.rb

```diff
@@ -29,21 +29,4 @@
     it { expect(@ticker.timestamp).to_not be nil }
     it { expect(@ticker.payload).to_not be nil }
   end
-
```

Comment:
@Swingcloud 

1. they should be in 2 different PRs (but it's fine this time)
2. did you remove the hack in the actual code?